File: data/code_pairs/fortran/292.f90
module modreport
  implicit none
  integer :: tests, nfailed, npassed
  integer :: reportfileu
  character(len=256) :: reportfilename
  character(len=256) :: testplan_name
  character(len=256) :: tdirectory
  character(len=20) :: testunitname, inputf, outputf
end module modreport

File: data/code_pairs/cpp/292.cpp
#ifndef MODREPORT_H
#define MODREPORT_H

#include <string>

class ModReport {
public:
    int tests = 0;
    int nfailed = 0;
    int npassed = 0;
    int reportfileu = 0;
    std::string reportfilename = "";
    std::string testplan_name = "";
    std::string tdirectory = "";
    std::string testunitname = "";
    std::string inputf = "";
    std::string outputf = "";

    // Constructor and Destructor
    ModReport() = default;
    ~ModReport() = default;
};

#endif // MODREPORT_H